content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -99,11 +99,13 @@ limitation and compatibility rules will change.
 
 #### Pause Image
 
-Microsoft maintains a Windows pause infrastructure container at
-`mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
 Kubernetes maintains a multi-architecture image `k8s.gcr.io/pause:3.5` that
 supports Linux as well as Windows.
 
+Microsoft maintains a Windows pause infrastructure container at
+`mcr.microsoft.com/oss/kubernetes/pause:3.5`.
+This image is built from the same source as the Kubernetes maintained image but all of the Windows binaries are (authenticode signed)[https://docs.microsoft.com/en-us/windows-hardware/drivers/install/authenticode] by Microsoft.
```

Comment:
We also don't build other archs such as `"architecture": "s390x",`

Reply:
I updated this to say the mcr image only container linux/windows amd64 images PTAL

Reply:
The link syntax is wrong, it should be `[authenticode signed](https://blahblah)`.

Reply:
whoops!